index/scorch/snapshot_index.go

```diff
@@ -744,6 +748,89 @@ func (i *IndexSnapshot) reClaimableDocsRatio() float64 {
 	return 0
 }
 
+func copyFile(src, dst string) (int64, error) {
+	sourceFileStat, err := os.Stat(src)
+	if err != nil {
+		return 0, err
+	}
+
+	if !sourceFileStat.Mode().IsRegular() {
+		return 0, fmt.Errorf("%s is not a regular file", src)
+	}
+
+	source, err := os.Open(src)
+	if err != nil {
+		return 0, err
+	}
+	defer source.Close()
+
+	destination, err := os.Create(dst)
+	if err != nil {
+		return 0, err
+	}
+	defer destination.Close()
+	nBytes, err := io.Copy(destination, source)
```

Comment:
nBytes, err := io.Copy(destination, source) Vs return io.Copy(destination, source) ...perhaps..

Reply:
ok sure, i honestly copy/pasted this from some Go 101 copy file example.